src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/attributes/DisplayAttributeInputs.tsx

```diff
@@ -50,6 +53,25 @@ const DisplayAttributeInputs = ({
             }}
           />
         );
+      case "DateTime":
+        return (
+          <DateTimeField
```

Comment:
there is a new `DateTimeInput` component that can be used over this. In the future, we might add some validations to it.

Reply:
Done, I think!